.github/workflows/ci.yml

```diff
@@ -96,8 +101,6 @@ jobs:
 
       - name: Run affected for lint
         run: npx nx run-many -t=build --projects@tailwindcss/node,@tailwindcss/oxide && npx prettier --check . && npx nx affected -t=lint --exclude nextjs-playground,vite-playground,@tailwindcss/root
-        # Only lint on linux to avoid \r\n line ending errors
-        if: matrix.runner == 'ubuntu-latest'
```

Comment:
This might not be necessary with lint running on the windows launch template.

Reply:
The reason I kept it is because that's how the tailwind repo has it originally, they only run lint on ubuntu!